webapp/src/components/Vendor/decentraland/NFTTopbar/SelectedFilters/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { collectionAPI } from "../../../../../modules/vendor/decentraland"
+
+export async function getCollectionNameByAddress(address: string, onlyOnSale: boolean | undefined) {
+  const { data } = await collectionAPI.fetch({
+    contractAddress: address,
+    isOnSale: onlyOnSale
+  })
+
+  if (data.length === 0) {
+    return undefined
+  }
+
+  return data[0].name
```

Comment:
```suggestion
  return data[0]?.name
```
Just a suggestion, if you like this one, you can delete the `if (data.length === 0) {` from above since it will return `undefined` if the array has no length